Makefile

```diff
@@ -57,4 +57,4 @@ clean-all: clean-docker-buildcache clean-docker-containers
 test:
 	make start-lumigator-build
```

Comment:
While we're at it, does it make more sense for `make test` to use start-lumigator-build or, rather, local-up one, since it's the target geared towards local development?

Non-blocking, by the way.

Reply:
it's a great question. I think this is a broader conversation going on about how we do this and the current mix of unit & integration tests. I'll keep it as it is right now (to sort out all the other problems people are having) and let's open this in a different thread/ticket!